internal/batches/executor/run_steps.go

```diff
@@ -303,6 +303,11 @@ func executeSingleStep(
 		args = append(args, "--mount", fmt.Sprintf("type=bind,source=%s,target=%s,ro", source.Name(), target))
 	}
 
+	// mount any paths on the local system to the docker container
+	for _, mount := range step.Mount {
+		args = append(args, "--mount", fmt.Sprintf("type=bind,source=%s,target=%s,ro", mount.Path, mount.Mountpoint))
+	}
```

Comment:
Can we add a flag to `executionOpts` like `allowFileMounts` that we wire through the executor and coordinator layer, so that we can explicitly disable it in `batch_exec.go` (the entrypoint for SSBC?) Feels like a good safety measure, in case someone can still sneak a batch spec into the server that contains mounts (I think we currently don't reject them in the SSBC `createBatchSpecFromRaw` API, just in the editor UI)

Reply:
Yea, we are not rejecting `mount` in the API layer at all.